grepos/app/src/main/java/net/boondockradio/grepos/adapter/RepositoryAdapter.java

```diff
@@ -0,0 +1,125 @@
+package net.boondockradio.grepos.adapter;
+
+import net.boondockradio.grepos.R;
+import net.boondockradio.grepos.dto.Repository;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import java.util.List;
+
+public class RepositoryAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+    private enum  ViewType {
+        Progress(0),
```

Comment:
first time, I use `ordinal` but it is not good way as far as I know.
http://futurismo.biz/archives/2768#unnumbered-7